bootloader/src/pyi_pythonlib.c

```diff
@@ -50,50 +50,53 @@
 /*
  * Load the Python DLL, and get all of the necessary entry points
  */
+#define DLLNAME_LEN 64
```

Comment:
This is not a good idea IMHO, since #defines are processed be the pre-processor and thus global, not local to the functions.

Reply:
I agree -- this should:

1. Go at the top of this file to emphasize the global nature.
2. Be written as `#define DLLNAME_LEN (64)`, as good practice to defend against the subtleties of preprocessor expansion.

Reply:
Hope I made the change per your review.

I did not add a comment specific to Windows - as it would fail on any platform if not defined.